pkg/storaged/format-dialog.jsx

```diff
@@ -244,8 +244,8 @@ export function format_dialog(client, path, start, size, enable_dos_extended) {
     }
 
     var filesystem_options = [ ];
-    add_fsys("xfs", { value: "xfs", title: _("XFS - Red Hat Enterprise Linux 7 default") });
-    add_fsys("ext4", { value: "ext4", title: _("ext4 - Red Hat Enterprise Linux 6 default") });
+    add_fsys("xfs", { value: "xfs", title: _("XFS - Recommended default") });
+    add_fsys("ext4", { value: "ext4", title: _("ext4 - For low-end hardware with limited I/O performance") });
```

Comment:
Let's upper-case EXT4 here, for consistency.

Reply:
I would leave out any description for EXT4 and VFAT.  It should be enough to guide those people to a good default that don't know yet what they want.

Reply:
I can drop the description for ext4, and also capitalize it for consistency (while we are changing the string anyway). But it would then be the only one without a description/application recommendation, which also feels a bit weird?

Reply:
Updated.